src/Components/Common/Sidebar/Sidebar.tsx

```diff
@@ -58,11 +59,15 @@ const StatelessSidebar = ({
   const activeLinkRef = useRef<HTMLAnchorElement>(null);
   const [lastIndicatorPosition, setLastIndicatorPosition] = useState(0);
   const [isOverflowVisible, setOverflowVisisble] = useState(false);
+  const [moreItems, setMoreItems] = useState(false);
```

Comment:
Variable "moreItems" does not convey the significance of it's value, whether it is an array of items or a boolean that depicts that more items to be shown or not.

```suggestion
  const [moreItemsShown, setMoreItemsShown] = useState(false);
```